src/api/v3/locations.py

```diff
@@ -0,0 +1,68 @@
+from django.http import JsonResponse
+
+from cms.models import POI
+from ..decorators import json_response
+
+
+def transform_poi_translation(poi_translation):
+    """
+    Function to create a JSON from a single poi_translation object.
+
+    :param poi_translation: The poi translation object which should be converted
+    :type poi_translation: ~cms.models.pois.poi_translation.POITranslation
+
+    :return: return data necessary for API
+    :rtype: dict
+    """
+
+    # is there a better way to access this?
+    try:
+        icon_url = poi_translation.icon.url
+    except AttributeError:
+        icon_url = ""
```

Comment:
This is probably not what I am supposed to do, how should I access the url correctly?